releasenotes/notes/quantumcircuit-optional-registers-60f0b2832ec95239.yaml

```
@@ -0,0 +1,8 @@
+---
+deprecations:
+  - |
+    Accessing references from :class:`~qiskit.circuit.Qubit` and
+    :class:`~qiskit.circuit.Clbit` instances to their containing registers
+    via the :meth:`~qiskit.circuit.Qubit.register` or :meth:`~qiskit.circuit.Qubit.index`
```

```suggestion
    via the :attr:`~qiskit.circuit.Qubit.register` or :attr:`~qiskit.circuit.Qubit.index`
```

Only merge this if there are no other changes (to avoid ci churn), if not I can fix this in #6075

---

Fixed in afdffa3 .